system/HTTP/Response.php

```diff
@@ -247,6 +253,7 @@ public function __construct($config)
 		$this->cookiePath     = $config->cookiePath;
 		$this->cookieSecure   = $config->cookieSecure;
 		$this->cookieHTTPOnly = $config->cookieHTTPOnly;
+		$this->cookieSameSite = $config->cookieSameSite;
```

Comment:
need default value fallback:

```suggestion
              $this->cookieSameSite = $config->cookieSameSite ?? '';
```